### PR TITLE
Reconcile text functions docs

### PR DESCRIPTION
**Pull request by diana-macrometa**
- Minor format and text changes, mostly in examples
- Removed redundant file

### CODE REVIEW

**Comment by diana-macrometa** [comment]
@StoyanTc or @ashish-macrometa, will you be able to review this soon?

File: docs/cep/query-guide/functions/sinkmapper/text.md [hunk 1]
```diff
@@ -2,69 +2,93 @@
...
+| delimiter              | This parameter specifies how events are separated when a grouped event is received. This must be a whole line and not a single character.    | \~\~\~\~\~\~\~\~\~\~ | STRING              | Yes      | No      |
+| new.line.character     | This attribute indicates the new line character of the event that is expected to be received. This is used mostly when communication between 2 types of operating systems is expected. For example, Linux uses `\n` whereas Windows uses `\r\n` as the end of line character. | \\n     | STRING              | Yes      | No      |
+| mustache.enabled       | If this parameter is set to `true`, then mustache mapping gets enabled for custom text mapping. | false    | BOOL                | Yes      | No      |
 
 ## Example 1
 
     CREATE SINK FooStream WITH (type='inMemory', topic='stock', map.type='text') (symbol string, price float, volume long);
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/sinkmapper/text.md]: Let's replace `inMemory` by `stream`.

File: docs/cep/query-guide/functions/sinkmapper/text.md [hunk 2]
```diff
@@ -2,69 +2,93 @@
...
-as follows: symbol:"gdn", price:55.6, volume:100
+This query performs a default text input mapping. The expected output is:
+
+    symbol:"GDN",
+    price:55.6,
+    volume:100
 
 ## Example 2
 
     CREATE SINK FooStream WITH (type='inMemory', topic='stock', map.type='text', map.event.grouping.enabled='true') (symbol string, price float, volume long);
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/sinkmapper/text.md]: Let's replace `inMemory` by `stream`.

**Review by StoyanTc — APPROVED** [review]

File: docs/cep/query-guide/functions/sinkmapper/text.md (hunk 2 above)
Comment (diana-macrometa) [reply docs/cep/query-guide/functions/sinkmapper/text.md]: Should I just do this everywhere in the docs? Or is it on a case-by-case basis?

Comment (StoyanTc) [reply docs/cep/query-guide/functions/sinkmapper/text.md]: Yes, this could be globally replaced.

Comment (diana-macrometa) [reply docs/cep/query-guide/functions/sinkmapper/text.md]: Then I shall do so in the JSON functions PR.

**Pull request merged**